ligo/src/types.mligo

```diff
@@ -0,0 +1,230 @@
+// Corresponds to Types.hs module
+
+// -- FA2 types -- //
+
+#if !TYPES_H
+#define TYPES_H
+
+type token_id = nat
+type token_metadata =
+  [@layout:comb]
+  { token_id : token_id
+  ; symbol : string
+  ; name : string
+  ; decimals : nat
+  ; extras : (string, string) map
+  }
+
+type operator =
+  { owner : address
+  ; operator : address
+  }
```

Comment:
It seems to be missing a `token_id` field, at least compared to its Lorentz counterpart.

Reply:
Could you point me to where do you see that? I can't find it :/

Reply:
Oh you are right, there isn't one.

My bad, I got confused with `operator_param`,